amd_openvx/openvx/ago/ago_kernel_api.cpp

```diff
@@ -22371,8 +22371,22 @@ int agoKernel_WeightedAverage_U8_U8_U8(AgoNode * node, AgoKernelCommand cmd)
     }
 #if ENABLE_OPENCL
     else if (cmd == ago_kernel_cmd_opencl_codegen) {
-        // TBD: not implemented yet
-        status = VX_ERROR_NOT_SUPPORTED;
+        status = VX_SUCCESS;
+        node->opencl_type = NODE_OPENCL_TYPE_REG2REG;
+        char textBuffer[2048];
+        sprintf(textBuffer, OPENCL_FORMAT(
+            "void %s (U8x8 * p0, U8x8 p1, float alpha, U8x8 p2)\n"
+            "{\n"
+            "   U8x8 r;\n"
+            "   float invAlpha = (float)1 - alpha;\n"
+            "   float4 alpha_f4 = make_float4(alpha, alpha, alpha, alpha);\n"
```

Comment:
in this line and the line below, please remove make_float4 function and instead of calling it just use (float4).